Formula/vapoursynth.rb

```diff
@@ -11,36 +11,39 @@ class Vapoursynth < Formula
     sha256 "90247e7b2a1fa10587512449d822ea548846ee904ffd26a0dace3e7944de61d5" => :mavericks
   end
 
-  # issue with upstream: https://github.com/vapoursynth/vapoursynth/issues/201
-  patch :DATA
-
   needs :cxx11
-  depends_on "pkg-config" => :build
+
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
   depends_on "yasm" => :build
-  depends_on :python3
 
-  depends_on "zimg"
-  depends_on "tesseract"
   depends_on "libass"
+  depends_on :python3
+  depends_on "tesseract"
+  depends_on "zimg"
 
-  resource "cython" do
-    url "https://pypi.python.org/packages/source/C/Cython/Cython-0.21.2.tar.gz"
-    sha256 "b01af23102143515e6138a4d5e185c2cfa588e0df61c0827de4257bac3393679"
+  resource "Cython" do
+    url "https://files.pythonhosted.org/packages/b1/51/bd5ef7dff3ae02a2c6047aa18d3d06df2fb8a40b00e938e7ea2f75544cac/Cython-0.24.tar.gz"
+    sha256 "6de44d8c482128efc12334641347a9c3e5098d807dd3c69e867fa8f84ec2a3f1"
   end
 
   def install
+    py3_cfg_prefix = Utils.popen_read("python3-config", "--prefix").chomp
```

Comment:
These two lines theoretically shouldn't actually be necessary any more. I symlinked both the `python` and `python3` pkg-config files to a sane location a fair while back now.